lib/vanagon/cli/list.rb

```diff
@@ -0,0 +1,65 @@
+require 'docopt'
+
+class Vanagon
+  class CLI
+    class List < Vanagon::CLI
+      DOCUMENTATION = <<~DOCOPT.freeze
+        Usage:
+          list [options]
+
+        Options:
+          -h, --help                       Display help
+          -c, --configdir DIRECTORY        Configuration directory [default: #{Dir.pwd}/configs]
+          -l, --platforms                  Display a list of platforms
+          -r, --projects                   Display a list of projects
+          -s, --space                      Displays the list as space separated
+      DOCOPT
+
+      def parse(argv)
+        Docopt.docopt(DOCUMENTATION, { argv: argv })
+      rescue Docopt::Exit => e
+        puts e.message
+        exit 1
+      end
+
+      def output(list, space)
+        if space
+          return list.join(' ')
+        end
+        return list
+      end
+
+      def run(options) # rubocop:disable Metrics/AbcSize
+        path = options[:configdir] || File.join(Dir.pwd, "configs")
```

Comment:
for some reason, `options[:configdir]` is resolving as false when no argument is passed. Which is strange because when I `puts options`, it shows `{:configdir =>"/Users/jacklynkinsler/puppet/bolt-vanagon/configs"}`